Detectors/ITSMFT/ITS/workflow/src/TrackerSpec.cxx

```diff
@@ -252,17 +252,14 @@ void TrackerDPL::run(ProcessingContext& pc)
         auto& trc{tracks[iTrk]};
         trc.setFirstClusterEntry(allClusIdx.size()); // before adding tracks, create final cluster indices
         int ncl = trc.getNumberOfClusters(), nclf = 0;
-        uint8_t patt = 0;
```

Comment:
@shahor02 please notice that I removed this part as the pattern is already during the tracking step: https://github.com/AliceO2Group/AliceO2/blob/0dad46d7a140af2e44fbd97b496d58377483b63e/Detectors/ITSMFT/ITS/tracking/src/Tracker.cxx#L287

These line were actually resetting the part of the `mPattern` reserved for MC information.